easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.53.5-foss-2017b.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'ConfigureMake'
+
+name = 'GObject-Introspection'
+version = '1.53.5'
+
+homepage = 'https://wiki.gnome.org/GObjectIntrospection/'
+description = """GObject introspection is a middleware layer between C libraries
+ (using GObject) and language bindings. The C library can be scanned at
+ compile time and generate a metadata file, in addition to the actual
+ native C library. Then at runtime, language bindings can read this
+ metadata and automatically provide bindings to call into the C library."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('Python', '2.7.14'),
```

Comment:
@Helios07 If there's a runtime dependency on Python, we usually indicate that via a `versionsuffix`, see https://github.com/easybuilders/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.53.5-intel-2017a-Python-3.6.1.eb#L5